From 321c196a0c2e8f8d998dd5c2821579acdb951e0f Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 21 Jun 2017 15:45:12 +0300
Subject: [PATCH 1723/2241] tools: nas: raid group_thread_cnt support

set the thread count as the cpu count, this improves raid performance

Change-Id: I12f9aa2dd06a7e2b736cb7b4e929aa24bd514c11
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40670
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 tools/nas/nas_init.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/nas/nas_init.sh b/tools/nas/nas_init.sh
index 29ab701..63aa360 100755
--- a/tools/nas/nas_init.sh
+++ b/tools/nas/nas_init.sh
@@ -610,6 +610,8 @@ else # RAID TOPOLOGY
 	echo -e 2048 > /sys/block/md0/queue/read_ahead_kb
 	echo 100000 > /sys/block/md0/md/sync_speed_min
 
+	echo $CPU_COUNT > /sys/block/md0/md/group_thread_cnt
+
 	set +o verbose
 	echo -ne "[Done]\n"
 fi
-- 
2.7.4

